azure-pipelines.yml

```diff
@@ -25,7 +25,8 @@ pr:
 jobs:
   - job: "Build_And_Test_Java_Worker_Windows"
     pool:
-      vmImage: 'windows-2019'
+      name: '1ES-Hosted-AzFunc'
+      vmImage: 'MMS2019TLS'  #for Windows2019 or MMSUbuntu20.04TLS for ubuntu
```

Comment:
> #for Windows2019 or MMSUbuntu20.04TLS for ubuntu

@eliaslopezgt Thanks! What about `windows-latest` ? The [dotnet worker build uses windows-latest](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/src/ci.yml#L21). Would it be `MMS2019TLS` for that as well?

Reply:
sent you a PM.